values.yaml

```diff
@@ -596,6 +596,10 @@ server:
   # Enables a headless service to be used by the Vault Statefulset
   service:
     enabled: true
+    active:
+      enabled: true
+    standby:
+      enabled: true
```

Comment:
Alternatively I guess we could pop out the active and standby service options into their own sections, though that would be a lot of duplication.

Reply:
Yeah if there are no objections, I would prefer to take a very small incremental step on this, as it already feels very granular to individually turn the services on/off. Perhaps it will make sense to split it out if we start getting a lot of feature requests to change more about each individual service in the future though.